src/main/java/com/cdevents/java/sdk/CDEventEnums.java

```diff
@@ -0,0 +1,66 @@
+package com.cdevents.java.sdk;

```

Comment:
Shouldn't this be dev.cdevents to be consistent with the domain name we own?

Reply:
yes, renamed package name to `dev.cdevents` as suggested.

Reply:
Thank you! 🙇